public/6.html

```diff
@@ -53,14 +53,14 @@
 function removeColorFromList(color) {
   $('.colorList').find('[data-id="'+color.id+'"]').remove();
   if ($('.colorList li').length === 0)  {
-    $('html').css('background', 'transparent');
+    $(document.body).css('background', 'transparent');
   }
 }
 function updateColorOnList(color) {
   $('.colorList').find('[data-id="'+color.id+'"]')
     .text(color.name+' (id: '+color.id+')');
   if (color.id === lastColorId) {
-    $('html').css('background', color.name);
+    $(document.body).css('background', color.name);
   }
 }
```

Comment:
can you send a separate pull request for these changes, and keep this one to the Opera change only?